bullet_train/lib/bullet_train/core_ext/callbacks_instrumentation/backport.rb

```diff
@@ -0,0 +1,118 @@
+return if ActiveSupport::VERSION.to_s.start_with?("7.2")
+
+# Backports a few commits from Rails 7.2 to make our debugging extension easier:
+#
+# - https://github.com/rails/rails/commit/6c5a042824fdaf630ebaa7fb293b8c2543f20d00
+# - https://github.com/rails/rails/commit/42ad4d6b0bfc14e16050f9945a8b2ac5c87c294f
```

Comment:
We need a few commits to make our instrumentation a little easier, so we backport these commits …except if users are running on Rails 7.2 where they already are.